src/main/kotlin/com/example/goodtripserver/trip/route/TripController.kt

```diff
@@ -0,0 +1,85 @@
+package com.example.goodtripserver.trip.route
+
+import com.example.goodtripserver.trip.model.AddCountryRequest
+import com.example.goodtripserver.trip.model.AddNoteRequest
+import com.example.goodtripserver.trip.model.AddTripRequest
+import com.example.goodtripserver.trip.service.TripService
+import com.goodtrip.goodtripserver.database.models.Trip
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseBody
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/trip")
+class TripController {
+
+    @Autowired
+    lateinit var tripService: TripService
```

Comment:
why we need lateinit? 
tip: we know that service should be instances only once, therefore you can just make this: 
```
class TripConstroller { 
private final tripService: TripService; 
} 
```

@Autowired basically is deprecated for final fields, but since it is final, we have to specify it in the constructor, so the injection would be a constructor type, so simplification you can just specify the @RequireAllArgsConstructor from lombook which will create a constructor with required final fields and after that if should work the same